couchbase/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # CHANGELOG - couchbase
 
-0.1.0/ Unreleased
+1.0.0/ Unreleased
 ==================
 
 ### Changes
 
-* [FEATURE] adds couchbase integration.
+* [FEATURE] adds couchbase integration to integrations-core.
```

Comment:
We should have either this as `Feature` or the camel case stuff all in caps. Just for consistency. Or maybe just label it `SDK` as I'm not sure it should really be considered a "feature".